ark/analysis/spatial_analysis.py

```diff
@@ -109,27 +115,38 @@ def calculate_cluster_spatial_enrichment(all_data, dist_matrices_dict, included_
-    cell types, looking for both positive and negative enrichment. Uses bootstrapping to permute cell labels randomly.
+    """Spatial enrichment analysis based on cell phenotypes to find significant interactions
+    between different cell types, looking for both positive and negative enrichment. Uses
+    bootstrapping to permute cell labels randomly.
 
     Args:
-        all_data: data including fovs, cell labels, and
-            cell expression matrix for all markers
-        dist_matrices_dict: A dictionary that contains a cells x cells matrix with the euclidian
-            distance between centers of corresponding cells for every fov
-        included_fovs: patient labels to include in analysis. If argument is none, default is all labels used
-        bootstrap_num: number of permutations for bootstrap. Default is 1000
-        dist_lim: cell proximity threshold. Default is 100
-        fov_col (string): column with the cell fovs (Default is SampleID)
-        cluster_name_col (string): column with the cell types (Default is cell_type)
-        cluster_id_col (string): column with the cell phenotype IDs (Default is FlowSOM_ID)
-        cell_label_col (string): column with the cell labels (Default is cellLabelInImage)
+        all_data (xarray.DataArray):
+            data including fovs, cell labels, and cell expression matrix for all markers
+        dist_matrices_dict (dict):
+            A dictionary that contains a cells x cells matrix with the euclidian distance between
+            centers of corresponding cells for every fov
+        included_fovs (list):
+            patient labels to include in analysis. If argument is none, default is all labels used
+        bootstrap_num (int):
+            number of permutations for bootstrap. Default is 1000
+        dist_lim (int):
+            cell proximity threshold. Default is 100
+        fov_col (str):
+            column with the cell fovs (Default is SampleID)
+        cluster_name_col (str):
+            column with the cell types (Default is cell_type)
+        cluster_id_col (str):
+            column with the cell phenotype IDs (Default is FlowSOM_ID)
+        cell_label_col (str):
+            column with the cell labels (Default is cellLabelInImage)
+
     Returns:
-        values (list): a list with each element consisting of a tuple of
-            closenum and closenumrand for each point included in the analysis
-        stats (xarray): an Xarray with dimensions (fovs, stats, number of channels, number of channels) The included stats
-            variables are:
-            z, muhat, sigmahat, p, h, adj_p, and cluster_names for each point in the analysis"""
+         - a list with each element consisting of a tuple of closenum and closenumrand for each
+         point included in the analysis
+         - an Xarray with dimensions (fovs, stats, number of channels, number of channels). The
```

Comment:
I think we can leave xarray lowercase? Probably we would say pandas array, not Pandas array. But I could be convinced.